sequences/convert.go

```diff
@@ -16,6 +16,18 @@ const EOL = "\r\n"
 
 var generatedByRegexp = regexp.MustCompile("; generated by PrusaSlicer .+( on .+)")
 
+func handleCoolingModuleGCodeInsert(materialCoolingModulePercentage int) string {
+	output := ""
+	if materialCoolingModulePercentage <= 0 {
+		output += EOL + "; Cooling Module fan off" + EOL
```

Comment:
Also, this comment is the KISSlicer style of comment – we should be matching Slic3r output here, i.e.

```gcode
M106 P10 S0 ; disable cooling module
```